lib/perl/Genome/Model/Tools/Joinx/VcfMerge.pm

```diff
@@ -168,6 +173,9 @@ sub _resolve_flags {
     if ($self->merge_samples) {
         $flags .= " -s";
     }
+    if ($self->allow_same_file) {
```

Comment:
You're right. I'm checking for version compatibility in Genome/VariantReporting/Suite/Vep/Run.pm but not here. Will add a check.